addons/delivery/models/delivery_carrier.py

```diff
@@ -108,6 +108,14 @@ def _match_address(self, partner):
             return False
         return True
 
+    def _get_amazon_carrier_name(self):
```

Comment:
When we said to change the name, it's because now, it doesn't send back the name of the carrier, so `_get_carrier_name` was wrong, as it now return the `delivery_type`.
I may be wrong, but the word "Amazon" should still not be present in any delivery module IMO.

Reply:
I see, so I'll change it to `_get_carrier_type`.